src/client/components/fleet-context.tsx

```diff
@@ -24,10 +28,25 @@ const ProviderWithJWT = ({ children }) => {
         (err) => err && console.warn('Failed to subscribe', err));
     }, [mqttSync]);
 
-  const fleet = {};
-  _.forEach(data?.[transitiveId], (device, id) =>
-    fleet[id] = mergeVersions(device['@transitive-robotics']['_robot-agent'])
-  );
+  const fleet = [];
+  _.forEach(data?.[transitiveId], (device, id) => {
+    const device_data = mergeVersions(device['@transitive-robotics']['_robot-agent']);
+    const capabilities = [];
+    if (device_data?.status?.runningPackages && device_data?.status?.runningPackages['@transitive-robotics']) {
+      capabilities.push(..._.map(device_data?.status?.runningPackages['@transitive-robotics'], (v, k) => k));
+    }
+    fleet.push(new Device(
+      id,
+      device_data?.info?.os?.hostname || id,
+      device_data?.info?.os?.lsb?.Description || 'Unknown',
+      device_data?.status?.heartbeat || new Date(),
+      capabilities,
+      Robot
+    ));
+  });
```

Comment:
To tell whether a capability is running you need to also verify that at least one of the values in the versions-object is truthy. And we can also make this a bit more compact using `.map` instead of `.forEach`.
```js
  const fleet = _.map(data?.[transitiveId], (device, id) => {
    const device_data = mergeVersions(device['@transitive-robotics']['_robot-agent']);
    const running = device_data?.status?.runningPackages?.['@transitive-robotics'];
    const capabilities = running ? Object.keys(_.pickBy(running,
        versions => Object.values(versions).some(Boolean))) : [];

    return new Device(
      id,
      device_data?.info?.os?.hostname || id,
      device_data?.info?.os?.lsb?.Description || 'Unknown',
      device_data?.status?.heartbeat || new Date(),
      capabilities,
      Robot
    );
  });
```

Reply:
addresed, also added:
![image](https://github.com/user-attachments/assets/9fb423ad-8778-4056-93b9-5ec3689c082e)